src/scene/mesh-instance.js

```diff
@@ -80,6 +81,38 @@ class InstancingData {
     }
 }
 
+/**
+ * Internal data structure used to store data used by indirect rendering.
+ *
+ * @ignore
+ */
+class IndirectData {
+    /**
+     * A map of camera components to their corresponding slot in the indirect buffer.
+     *
+     * @type {Map<CameraComponent, number>}
+     */
+    map = new Map();
+
+    /**
+     * An array of 4 integers used to store mesh metadata needed for indirect rendering.
+     *
+     * @type {Int32Array}
+     */
+    meshMetaData = new Int32Array(4);
+
+    /**
+     * Retrieves the slot in the indirect buffer for a specific camera.
+     *
+     * @param {CameraComponent|null} camera - The camera component to retrieve the slot for, or null
```

Comment:
I know there are other places in the scene layer that import from the framework layers...but here's a new one!

Reply:
Yep, I'm stuck with it here. Camera is not public so I cannot use that, and don't see a good way around it.